packages/docusaurus-theme-classic/src/theme/DocCard/index.tsx

```diff
@@ -49,7 +48,11 @@ function CardLayout({
   icon: ReactNode;
   title: string;
   description?: string;
-}): JSX.Element {
+}): JSX.Element | null {
+  if (!href) {
```

Comment:
can you filter items in DocCardList instead? the card shouldn't be rendered at all if we don't wait it to appear here